### DISCUSSION TITLE
terraform-aws-service-catalog support for EKS 1.26

### DISCUSSION THREAD

**Discussion by jeffreymlewis** [gruntwork-io/knowledge-base#742] [Help] [answered]
Amazon has rev'ed EKS twice in the last few months, and are up to EKS 1.27 as of May 24th.

I see Gruntwork has updated [terraform-aws-eks](https://github.com/gruntwork-io/terraform-aws-eks) to support EKS 1.26 (in version `v0.59.0`), but [terraform-aws-service-catalog](https://github.com/gruntwork-io/terraform-aws-service-catalog) still needs to be upgraded. ([Example](https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/modules/services/eks-core-services/main.tf#L243))

When can we expect `terraform-aws-service-catalog` to support EKS 1.26 ?

Thanks,
Jeffrey

---

<ins datetime="2023-06-23T17:13:47Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110296">Tracked in ticket #110296</a></p>
</ins>

**Comment by josh-padnick** [comment]
@ryehowell @pras111gw Could you chime in on the specific timeline here?

Jeffrey, on the product side, I can tell you that now that Patcher is becoming more mature, we're turning our attention to writing patches and we'll be focusing specifically on EKS upgrades so that we can make this very process both timely and streamlined.

cc: @techpink

**Reply by pras111gw** [reply, marked as answer]
Catalog update is in flight for 1.26 and it should release early next week. I have started reviewing 1.27 as well and it should be released in couple of weeks.

**Reply by josh-padnick** [reply]
Fixed in https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/v0.104.14!
